lib/src/transformer/impl_generation.dart

```diff
@@ -617,6 +608,24 @@ class ImplGenerator {
         sourceFile.location(typedMap.node.leftBracket.end),
         staticVariablesImpl
     );
+
+    final name = typedMap.node.name.name;
+    final metaClassName = '$generatedPrefix${name}Meta';
+    final metaStructName = type == AccessorType.props
+        ? 'PropsMeta'
+        : 'StateMeta';
+    final metaInstanceName = '${publicGeneratedPrefix}metaFor$name'; // FIXME validate boilerplate to avoid typos
+     var output = new StringBuffer();
+    output.writeln('/// A class that allows us to reuse generated code from the accessors class.');
+    output.writeln('/// This is only used by other generated code, and can be simplified if needed.');
+    output.writeln('class $metaClassName {');
+    output.writeln(staticVariablesImpl);
+    output.writeln('}');
+    output.writeln('const $metaStructName $metaInstanceName = const $metaStructName(');
+    output.writeln('  fields: $metaClassName.$constantListName,'); // todo don't include generated prefix
+    output.writeln('  keys: $metaClassName.$keyListName,'); // todo don't include generated prefix
```

Comment:
These todos were also from the original implementation, but I don't understand them. @greglittlefield-wf is this saying that we should drop the `$` prefix from `$props`/`$state`/`$propKeys`/`$stateKeys`?

Reply:
That's a good question... whatever it is, I don't think it's super important, though; this code looks good to me as-is.

It might be referring to the fact that the variables are just copied from the generated accessors class to the `$metaClassName` class, prefix and all, when that's unnecessary.

Or, it could be referring to something related to the simplification noted in the generated doc comment on 620.

Reply:
> I don't think it's super important, though

For more context, I usually use "fixme" comments if it's important, not todos 😄